core/src/main/java/kafka/log/remote/quota/RLMQuotaManager.java

```diff
@@ -77,16 +78,16 @@ public void updateQuota(Quota newQuota) {
         }
     }
 
-    public boolean isQuotaExceeded() {
+    public long checkQuotaAndGetThrottleTimeMs() {
```

Comment:
nit: The method name can be just `getThrottleTimeMs`. WDYT?

Reply:
Done